src/DurableTask.AzureStorage/Partitioning/TablePartitionManager.cs

```
@@ -0,0 +1,795 @@
+                        this.settings.WorkerId,
+                        "",
+                        $"Error in stopping partition manager : {exception}");
+                }
+                await Task.Delay(timeToSleep);
+            };
+
+            this.settings.Logger.PartitionManagerInfo(
+                this.storageAccountName,
+                this.settings.TaskHubName,
+                this.settings.WorkerId,
+                "",
+                $"Successfully released all ownership leases for shutdown.");
+        }
+
+        async Task IPartitionManager.CreateLeaseStore()
+        {
+            try
+            {
+                await this.partitionTable.CreateIfNotExistsAsync();
+            }
+            catch (Exception exception)
+            {
+                this.settings.Logger.PartitionManagerError(
+                    this.storageAccountName,
+                    this.settings.TaskHubName,
+                    this.settings.WorkerId,
+                    "",
+                    $"Failed to create partition table: {exception}");
+            }
+        }
+
+        async Task IPartitionManager.CreateLease(string leaseName)
+        {
+            try
+            {
+                var lease = new TableLease()
+                {
+                    PartitionKey = "",
+                    RowKey = leaseName
+                };
+                await this.partitionTable.AddEntityAsync(lease);
+            }
+            catch (RequestFailedException e) when (e.Status == 409 /* The specified entity already exists. */)
+            {
+                this.settings.Logger.PartitionManagerInfo(
+                    this.storageAccountName,
+                    this.settings.TaskHubName,
+                    this.settings.WorkerId,
+                    leaseName,
+                    $"The partition {leaseName} already exists in the table.");
+            }
+            //Eat any other exceptions.
```

Can you explain why it's safe to eat any other exceptions?

---

As the other reply.